corehq/apps/reports/standard/monitoring.py

```diff
@@ -969,8 +969,8 @@ def generate_chart(cls, data, width=950, height=300, timezone="UTC"):
             sizes.extend([1] * 24)
         chart.add_data(sizes)
 
-        chart.set_axis_labels('x', [''] + [(str(h)+'h') for h in range(24)] + [''])
-        chart.set_axis_labels('x', [' ', 'Time ({})'.format(timezone), ' '])
+        chart.set_axis_labels('x', [''] + [(str(h) + 'h') for h in range(24)] + [''])
```

Comment:
Hmm, good point. I guess the 13h isn't convention outside of europe. Would it be clearer if I added `:00` instead of the `h`? 
i.e. 0:00, 13:00, etc.